src/main/java/de/retest/recheck/cli/subcommands/Ignore.java

```diff
@@ -84,10 +92,16 @@ public void run() {
 				saveRecheckIgnore();
 			} catch ( final Exception e ) {
 				ErrorHandler.handle( e );
+				exitCode = SOFTWARE;
```

Comment:
Maybe put before the ErrorHandler so the code is changed in any case.

Reply:
Also in all other cases.

Reply:
Good catch! I changed it.